infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/LoggingConfigurator.java

```diff
@@ -246,12 +250,9 @@ private static LoggerConfig setUpValidatorLogger(final Appender appender) {
   }
 
   private static LoggerConfig setUpP2pLogger(final Appender appender) {
-    // Don't disable p2p error logs unless the root log level disables error.
-    final Level p2pLogLevel =
-        INCLUDE_P2P_WARNINGS || ROOT_LOG_LEVEL.isMoreSpecificThan(Level.ERROR)
-            ? ROOT_LOG_LEVEL
-            : Level.ERROR;
+    final Level p2pLogLevel = INCLUDE_P2P_WARNINGS ? Level.INFO : Level.OFF;
     final LoggerConfig logger = new LoggerConfig(P2P_LOGGER_NAME, p2pLogLevel, true);
+    StatusLogger.getLogger().info("Include P2P warnings set to: {}", INCLUDE_P2P_WARNINGS);
     logger.addAppender(appender, ROOT_LOG_LEVEL, null);
     return logger;
```

Comment:
I think we should just be removing this whole section.  The logger doesn't need to be treated specially anymore because we're toggling whether to log or not in `P2PLogger` itself now.